src/main/java/com/coherentsolutions/training/auto/web/pashkovskaya/pages/HomePageAuthorizedUser.java

```diff
@@ -0,0 +1,32 @@
+package com.coherentsolutions.training.auto.web.pashkovskaya.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePageAuthorizedUser extends BasePage{
+    @FindBy(xpath = "//div[@class = 'panel header'] //button[@class = 'action switch']")
+    public WebElement loggedInUserLabel;
+    @FindBy(linkText = "My Account")
+    public WebElement myAccountPage;
+    @FindBy(linkText = "Women")
+    public WebElement womenLink;
+
+    public HomePageAuthorizedUser(WebDriver driver) {
+        super(driver);
+    }
+
+    public MyAccountPage openMyAccountPage() throws InterruptedException {
+        Thread.sleep(2500);
+        loggedInUserLabel.click();
+        myAccountPage.click();
+
+        return new MyAccountPage(driver);
+    }
+
+    public WomenClothesPage openWomenClothesPage() {
```

Comment:
is there any difference between WomenClothesPage and other pages except content?

Reply:
As far as I understand, there is no difference only between the WomenClothesPage and MenClothesPage. I don't use MenClothesPage, so I didn't add it in my project